src/flb_pack.c

```diff
@@ -951,6 +956,11 @@ flb_sds_t flb_pack_msgpack_to_json_format(const char *data, uint64_t bytes,
             case FLB_PACK_JSON_DATE_EPOCH:
                 msgpack_pack_uint64(&tmp_pck, (long long unsigned)(tms.tm.tv_sec));
                 break;
+            case FLB_PACK_JSON_DATE_EPOCH_MS:
+                msgpack_pack_uint64(&tmp_pck,
+                                    (long long unsigned)(tms.tm.tv_sec) * 1000 +
```

Comment:
Could you create the function `flb_time_to_millisec` in `flb_time.c` and move that conversion there so it can be reused by others in the future? Also, please use uint64_t like `flb_time_to_nanosec`.

Reply:
@jeongki-kim Could you make these changes, please?

Reply:
Thank you for your review. Had I found this earlier, I'd have followed your suggestion.